services/license-store.js

```diff
@@ -0,0 +1,99 @@
+const assert = require('assert');
+
+class License {
```

Comment:
It feels a bit unintuitive to me to have this in the same file as `LicenseStore`. I guess it's because one can spare exporting it 🤔 Or does this have an underlying best practise pattern?

Reply:
No, there was no particular reason to not put it in a separate file other than me playing it simple while I was porting over the whole shebang 😀